CMSIS/Core/Include/cmsis_compiler.h

```diff
@@ -53,6 +53,15 @@
  */
 #elif defined ( __ICCARM__ )
 
+  /* CMSIS compiler control architecture macros */
```

Comment:
Anders Lundgren from IAR aims to include these defines into their `cmsis_iar.h`. He announce the release 8.10 for today which should contain the attached version of their header file. Please review the changes shipped by IAR. If the changes are working we can skip the defines here. Please update the PR accordingly.